pom.xml

```diff
@@ -481,7 +527,7 @@
                     <version>2.23.0</version>
                     <configuration>
                         <!--suppress MavenModelInspection -->
-                        <configFile>${rootdir}/etc/eclipse-formatter-config.xml</configFile>
+                        <configFile>/Users/carlesarnal/IdeaProjects/kroxylicious/etc/eclipse-formatter-config.xml</configFile>
```

Comment:
you probably don't want this.

Reply:
yup, leftover from local testing, removing...

Reply:
`${rootdir}` should be resolved when running the build. Did you have particular problems that meant you ended up replacing it?